apps/server/src/adapters/OscAdapter.ts

```diff
@@ -59,17 +60,29 @@ export class OscServer implements IAdapter {
           property,
           value,
         } satisfies ChangeOptions;
+      } else if (params.length) {
+        // transform params array into a nested object with args as the value
+        // /a/b/c/d 5 will become {a:{b:{c:{d: 5}}}}
+        transformedPayload = params.reduceRight(
+          (parm, key, index) => (index === params.length - 1 ? { [key]: args } : { [key]: parm }),
+          {},
+        );
       }
 
       try {
-        // we dont reply on OSC
-        dispatchFromAdapter(
+        const reply = dispatchFromAdapter(
           path,
           {
             payload: transformedPayload,
           },
           'osc',
         );
+        //We only send simple types back to the OSC client
+        if (typeof reply.payload === 'string' || typeof reply.payload === 'number') {
```

Comment:
Is there an issue we have in mind? we could force stringify it and send whatever... although I am unsure on how replies would work in OSC since the client has closed the connection after sending the request

Reply:
we cold send a stringifyed responds to the client but I think very few are ready to handle that
we could also unpack the object and translate it to a bundle OSC messages (this might actually be quite good)
it is just UDP so no connection is open. We are just throwing the reply back to the same port and they can pick it up if they want it

Reply:
I think the best is to skip feedback on UDP.

We have the integration output which should provide adequate feedback to changes in the application state